Formula/criterion.rb

```diff
@@ -21,6 +20,7 @@ class Criterion < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "debugbreak"
```

Comment:
> meson.build:144:13: ERROR: Dependency 'debugbreak' is required but not found.

I mean, it's there though.

Reply:
yeah, ran into the same issue, weird

Reply:
Could it be expecting the executable in standard `PATH` rather than the environment’s `PATH`? I’m not sure how meson.build works